demos/README.md

```diff
@@ -202,6 +206,29 @@ build binaries in Debug configuration. Run the appropriate version of the
 Microsoft Visual Studio and open the generated solution file from the `C:\Users\<username>\Documents\Intel\OpenVINO\omz_demos_build\Demos.sln`
 directory.
 
+You can also build the demo applications using `cmake --build` tool:
+1. Navigate to a directory that you have write access to and create a demos build directory. This example uses a directory named `build`:
+```
+md build
+```
+2. Go to the created directory:
+```
+cd build
+```
+3. Run CMake to generate project files:
+```
+cmake <open_model_zoo>/demos
```

Comment:
I think `-A x64` should be added here, because CMake defaults to 32-bit for Visual Studio generators prior to 2019.

Reply:
Added